writer/standard/image_option.go

```diff
@@ -2,10 +2,9 @@ package standard
 
 import (
 	"fmt"
+	"github.com/yeqown/go-qrcode/v2"
```

Comment:
Keeps imports in order of [std, library itself, third party]

Reply:
@yeqown fixed